Components/Login/loginController.js

```diff
@@ -0,0 +1,18 @@
+var app = angular.module("angularForm");
+app.controller("loginController", function($scope, $http, getdatafactory, $timeout, $localStorage) {
```

Comment:
getdatafactory is wrong. it should be camel case always
getdatafactory ==> getDataFactory